audiotools/core/ffmpeg.py

```diff
@@ -177,14 +183,15 @@ def load_from_file_with_ffmpeg(cls, audio_path: str, quiet: bool = True, **kwarg
             # We pad the file using the start time offset
             # in case it's an audio stream starting at some
             # offset in a video container.
-            pad = ffprobe_offset(audio_path)
-            # Don't pad files with discrepancies less than
+            pad, codec = ffprobe_offset_and_codec(audio_path)
+            
+            # For mp3s, don't pad files with discrepancies less than
             # 0.027s - it's likely due to codec latency.
             # The amount of latency introduced by mp3 is
             # 1152, which is 0.0261 44khz. So we
             # set the threshold here slightly above that.
             # Source: https://lame.sourceforge.io/tech-FAQ.txt.
-            if pad < 0.027:
+            if codec == 'mp3' and pad < 0.027:
```

Comment:
I'm still trying to get up to speed on all the context here, but is it worth linking this line and https://github.com/descriptinc/descript-web-v2/pull/22459/files via comments somehow? IIUC, they need to be the same logic, or there will be a minor offset between studio sound and something else (the rest of the project?)

Reply:
Added a comment so it's linked with the linear task!